komga-webui/src/locales/en.json

```diff
@@ -130,7 +130,7 @@
       "first_page": "First page",
       "fullscreen": "Enter/exit full screen",
       "last_page": "Last page",
-      "left_to_right": "Left to Right",
+      "left_to_right": "Left to Right →",
```

Comment:
why the icon?

Reply:
About 25% (some number anyway) of people have difficulty ascribing direction to "left" or "right". Helps cognition.